source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -17,6 +17,17 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - 4.4
+     - ✓
```

Comment:
Does 4.4. support all 5.1 features? If not, let's not add a checkmark here as that's what that indicates.